probe-rs-tools/src/bin/probe-rs/cmd/run.rs

```diff
@@ -146,6 +146,8 @@ pub struct SharedOptions {
 
     /// The format string to use when printing defmt encoded log messages from the target.
     ///
+    /// You can also use one of two presets: oneline (default) and full.
```

Comment:
Isn't there `twoline` now as well?

Reply:
I did have it in one of the first iterations, but decided that "full" is a better name. Code that decides looks like this:

```rust
        let format = match log_format {
            None | Some("oneline") => FormatterFormat::OneLine {
                with_location: show_location,
            },
            Some("full") => FormatterFormat::Default {
                with_location: show_location,
            },
            Some(format) => FormatterFormat::Custom(format),
        };
```

Reply:
NVM